erpnext/setup/doctype/company/company.json

```diff
@@ -1414,6 +1457,40 @@
    "unique": 0, 
    "width": "50%"
   }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_in_quick_entry": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "depends_on": "eval:doc.parent_company", 
+   "fetch_if_empty": 0, 
+   "fieldname": "ignore_root_company_validation", 
+   "fieldtype": "Check", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "Ignore Root Company Validation", 
```

Comment:
@deepeshgarg007 Very bad naming. How the users will know what is the "Root Company Validation". 
May be "Allow Account Creation against Child Company" will be a better name.